Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -1700,6 +1724,20 @@
   categories:
   - UplinkWearables
 
+- type: listing
+  id: UplinkHardsuitSyndieelite
+  name: Syndycate elite Modsuit 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)